### ISSUE TITLE
Copy my docker-machine config to another physical machine (client)?

### ISSUE THREAD

**Issue by appsolutegeek** [docker/machine#4304]
Hi,

I have docker-machine setup and working from a mac, and it has all my machines attached and working.  I would like to also use docker-machine (my config) from another mac.

What are my options ?  I know under the .docker/machine there are the certificates, do I need to copy these manually or is there some sort of process.

To summarise, I would like to have my current docker-machine setup (with all machines) available on another machine.

What is the best way to execute this ?

Thanks in advance.

**Comment by kaikuchn** [comment]
Yes, you need to copy the certificates and machines (everything in `~/.docker/machine/`) to the other client and then adjust the paths in the `config.json` files contained in the machine folders.
I.e. run `sed -i.bak s/oldpath/newpath/g ~/.docker/machine/machines/*/config.json`
